cli/cluster.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 Prometheus Team
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"context"
+	"errors"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+
+	"github.com/prometheus/alertmanager/cli/format"
+)
+
+const clusterHelp = `View cluster status.
+
+The amount of output is controlled by the output selection flag:
+	- Simple: Print just cluster status
+	- Extended: Print the peers and their names with cluster status
+	- Json: Print entire cluster status object as json
```

Comment:
I would remove this part since the `--output` flag is supported by all commands.